_data/nav.yml

```diff
@@ -166,6 +165,8 @@ About:
   - href: /about/contribute/
   - href: /about/community/
   - href: /about/whats-new/
+  - text: Monthly calls
+    href: /about/monthly-calls/
```

Comment:
It looks like the other pages get their text from the `title` on their yml file. Is there a benefit to defining `text` here?

Reply:
I just thought that "Past monthly calls" looked strange as a sidenav link, so I took the liberty of changing it to "Monthly calls". 

@sarah-sch Are you ok with having the sidenav link say "Monthly calls" and the page title be "Past monthly calls"? Or do you want the sidenav link to also read "Past monthly calls"? Or maybe we change the page title to "Monthly calls" since we also mention future monthly calls?

Reply:
Oh that's a good point! I definitely prefer "Monthly calls" over "Past monthly calls".

I also think there's value in having a general page for monthly calls. We could beef this page up with some additional information about monthly calls and what people can expect from them.

If we end up keeping the page title the same, I think its a good idea to update this text.

Reply:
@amyleadem - it should definitely be "Monthly calls" in the side nav. I think it makes sense to go ahead and change the page title too. Thanks!

Reply:
@sarah-sch I updated the page title to "Monthly calls" to match the sidenav link title. Can you take a look and hit "approve" if it looks good to you? 

Resolved in [6dac2b0](https://github.com/uswds/uswds-site/pull/2345/commits/6dac2b028e269f6803b8bbce852826a8227a833e)

Reply:
@amyleadem - the change looks good to me.